src/containers/Downloads/index.js

```diff
@@ -178,7 +194,12 @@ class Download extends Component {
         ) : (
           <div>
             <DownloadFileSummary summaryData={downloadFilesData} />
-            <DownloadDatasetSummary data={this.props.dataSet} />
+            <DownloadDatasetSummary
```

Comment:
Is it possible to use connect on `DownloadDatasetSummary` and send these properties directly into it? (`samplesBySpecies`, `totalSamples`, `totalExperiments` and `experimentCountBySpecies`) Instead of putting them on `Download` and passing them down

Reply:
Hmm it is possible, but I'm not sure it's necessary to connect it since the parent needs and has access to those props already.

Reply:
That's true, I didn't realize of those usages.